pkg/cri/store/image/image.go

```diff
@@ -206,7 +205,7 @@ func (s *store) add(img Image) error {
 		return nil
 	}
 	// Or else, merge and sort the references.
-	i.References = docker.Sort(util.MergeStringSlices(i.References, img.References))
+	i.References = util.MergeSortedStringSlices(i.References, img.References)
```

Comment:
Does it work? docker.Sort is having a special sorting order for refs.

https://github.com/containerd/containerd/blob/326cd0623eb99aa6c0fd6b1255add105377d789e/reference/docker/sort.go#L31

Reply:
Got it, we shouldn't cancel docker.Sort here